src/help.h

```diff
@@ -787,6 +787,11 @@ struct commandHelp {
     "Determine if a given value is a member of a set",
     3,
     "1.0.0" },
+    { "SMISMEMBER",
+    "key member [member ...]",
+    "Determine if one or more given values are members of a set",
```

Comment:
Is this really the correct description? This text (to me) sounds like you're returning true iff at least one is a member, and not `determine for each given value if it's a member of a set` (or similar).